src/js/store/factory/marks/Scene.ts

```diff
@@ -5,16 +5,17 @@ import {GroupMark} from 'vega-typings/types';
 export interface LyraSceneMark extends Omit<GroupMark, 'type'> {
   _id: number;
   _parent: number;
-  type: 'scene';
+  type: 'group';
```

Comment:
great change. since the type is now group (making the child interface identical to the original group interface), LyraSceneMark probably does not need to be its own interface anymore.

(i'll make this change)